buildconfig/pygame-stubs/sprite.pyi

```diff
@@ -72,58 +55,48 @@ class RenderClear(Group):
 
 class RenderUpdates(Group):
     def copy(self) -> RenderUpdates: ...
-    def draw(self, surface: Surface) -> List[Rect]: ...
+    def draw(self, surface: Surface) -> Sequence[Rect]: ...
```

Comment:
Maybe List is better here because it's a return type of List?

Reply:
fixed